package.json

```diff
@@ -1396,6 +1412,26 @@
                     "when": "view == aws.explorer && viewItem == awsEcrNode",
                     "group": "inline@1"
                 },
+                {
+                    "command": "aws.emrserverless.startApplication",
+                    "when": "view == aws.explorer && viewItem == awsEmrServerlessStoppedApplicationNode",
```

Comment:
changing the node id is a questionable pattern that we should probably avoid. can't the node detect its state and make a decision based on the current state instead of quasi-parameterizing things in package.json ?

Reply:
Noted - will look into that. Should definitely be doable, just curious about the user flow. I liked the idea of having separate "start/stop" commands and wasn't sure how to go about that otherwise.